plugins/utils/neoclip.nix

```diff
@@ -0,0 +1,240 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "neoclip";
+  originalName = "statuscol.nvim";
```

Comment:
```suggestion
  originalName = "nvim-neoclip.lua";
```

Reply:
...

Reply:
You've changed it to `"neoclip.nvim"`, but upstream is definitely using `"nvim-neoclip.lua"` in their repo name and README header: https://github.com/AckslD/nvim-neoclip.lua